File: MODUL_3/Py/PRAK305_2310817310014_MuhammadRizkiSaputra.py
DETIK = int(input(""))

HARI = DETIK // 86400
DETIK %= 86400

JAM = DETIK // 3600
DETIK %= 3600

MENIT = DETIK // 60
DETIK2 = DETIK % 60

hasil = f"{HARI} Hari " if HARI > 0 else ""
hasil += f"{JAM:02d}:{MENIT:02d}:{DETIK2:02d}"

print(hasil)

File: MODUL_3/C/PRAK305_2310817310014_MuhammadRizkiSaputra.c
#include <stdio.h>

int main() {
    int DETIK;

    printf("");
    scanf("%d", &DETIK);

    int HARI = DETIK / 86400;
    DETIK %= 86400;

    int JAM = DETIK / 3600;
    DETIK %= 3600;

    int MENIT = DETIK / 60;
    int DETIK2 = DETIK % 60;

    if (HARI > 0) {
        printf("%d Hari ", HARI);
    }

    printf("%.2d:%.2d:%.2d\n", JAM, MENIT, DETIK2);

    return 0;
}
